core/block/import/objectcreator.go

```diff
@@ -366,7 +365,7 @@ func (oc *ObjectCreator) resetState(ctx *session.Context, newID string, st *stat
 		}
 		commonOperations, ok := b.(basic.CommonOperations)
 		if !ok {
-			return fmt.Errorf("common operations is not allowed for this object")
```

Comment:
All errors from `resetState` are just  logged in above function. This log is also useless and it's not a error. It's a normal situation, when some object doesn't support CommonOperations